### PR TITLE
Word2vec basic show id and word together 

### PR DESCRIPTION
**Pull request by hunkim**
When I first read the output, I spent a lot of time to understand these numbers to match them. Now, words along with the ids clearly show what's going on in the sample data and prediction pairs.

[New output]: clearly shows the word ids and corresponding words
      Sample data [5239, 3084, 12, 6, 195, 2, 3137, 46, 59, 156] ['anarchism', 'originated', 'as', 'a', 'term', 'of', 'abuse', 'first', 'used', 'against']
      3084 originated -> 12 as
      3084 originated -> 5239 anarchism
      12 as -> 6 a
      12 as -> 3084 originated
      6 a -> 12 as
      6 a -> 195 term
      195 term -> 6 a
      195 term -> 2 of

[Old output]: No words for Sample data. Word ids and words are mixed, so it's very hard to read
      Sample data [5239, 3084, 12, 6, 195, 2, 3137, 46, 59, 156] ['anarchism', 'originated', 'as', 'a', 'term', 'of', 'abuse', 'first', 'used', 'against']
      3084 -> 5239
      originated -> anarchism
      3084 -> 12
      originated -> as
      12 -> 3084
      as -> originated
      12 -> 6
      as -> a
      6 -> 195
      a -> term
      6 -> 12
      a -> as
      195 -> 6
      term -> a
      195 -> 2
      term -> of

### CODE REVIEW

**Comment by vrv** [comment]
@tensorflow-jenkins: test this please

**Comment by gunan** [comment]
Can one of the admins verify this patch?

**Comment by hunkim** [comment]
@vrv Hmm. I am not sure where/how this BR brakes those tests. It was fine in my environment. I'll double check.

**Comment by vrv** [comment]
no, jenkins broke, we restarted it.  let me try again:  @tensorflow-jenkins test this please

**Comment by vrv** [comment]
Jenkins keeps breaking but python3/mac tests passed so this is syntactically fine.  Merging.
